modules/monitoring-about-creating-alerting-rules-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-creating-alerting-rules-for-user-defined-projects_{context}"]
+= About creating alerting rules for user-defined projects
+
+If you create alerting rules for a user-defined project, consider the following key behaviors and important limitations when you define the new rules:
+
+* You can only create new alerting rules for user-defined projects.
+You cannot create new alerting rules for core {product-title} monitoring projects.
+Core platform monitoring by default provides a set of alerting rules for these projects.
+
+* A user-defined alerting rule can include metrics exposed by its own project in addition to the default metrics from core platform monitoring. 
+You cannot include metrics from another user-defined project.
++
+For example, an alerting rule for the `ns1` user-defined project can use metrics exposed by the `ns1` project in addition to core platform metrics, such as CPU and memory metrics.
+However, the rule cannot include metrics from a different `ns2` user-defined project.
+
+* To reduce latency and to minimize the load on core platform monitoring components, you can add the `openshift.io/prometheus-rule-evaluation-scope: leaf-prometheus` label to a rule to force only the Prometheus instance deployed in the user-defined project to evaluate the alerting rule instead of the Thanos Ruler instance used by default platform monitoring.
```

Comment:
```suggestion
* To reduce latency and to minimize the load on core platform monitoring components, you can add the `openshift.io/prometheus-rule-evaluation-scope: leaf-prometheus` label to a rule to force only the Prometheus instance deployed in the `openshift-user-workload-monitoring` project to evaluate the alerting rule instead of the Thanos Ruler instance.
```

Reply:
changed